.goreleaser.yml

```diff
@@ -31,16 +31,18 @@ release:
   name_template: "{{ .Version }}"
 dockers:
   -
+    # Need to use goreleaser Pro to get podman support
```

Comment:
Should there be a ticket on this? I'm not sure, but raising a question.

Reply:
Sure I can add a ticket for this, will do this.

Reply:
https://vaporio.atlassian.net/browse/VIO-1931